changeLogs/index.ts

```diff
@@ -0,0 +1,11 @@
+import version0161 from './test0161.md';
+const version = process.env.version!.split('/')[0].replace(/[^0-9]/g, '');
+const versionMap = {
+  '0161': version0161,
+  '0167': version0161,
```

Comment:
mock data for test should not merge into master, just remove them after test passed

Reply:
Yes, you are right, will add form notice before merge into master